apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/database/VedtaksvurderingRepository.kt

```diff
@@ -348,15 +314,14 @@ class VedtaksvurderingRepository(private val datasource: DataSource) {
         },
         beregningsResultat = getJsonObject(5),
         vilkaarsResultat = getJsonObject(6),
-        kommerSoekerTilgodeResultat = getJsonObject(7),
-        vedtakFattet = getBoolean(8),
-        fnr = getString(10),
-        datoFattet = getTimestamp(11)?.toInstant(),
-        datoattestert = getTimestamp(12)?.toInstant(),
-        attestant = getString(13),
-        virkningsDato = getDate(14)?.toLocalDate(),
-        vedtakStatus = getString(15)?.let { VedtakStatus.valueOf(it) },
-        behandlingType = getString(17)?.let { BehandlingType.valueOf(it) }
+        vedtakFattet = getBoolean(7),
+        fnr = getString(9),
+        datoFattet = getTimestamp(10)?.toInstant(),
+        datoattestert = getTimestamp(11)?.toInstant(),
+        attestant = getString(12),
+        virkningsDato = getDate(13)?.toLocalDate(),
+        vedtakStatus = getString(14)?.let { VedtakStatus.valueOf(it) },
+        behandlingType = getString(16)?.let { BehandlingType.valueOf(it) }
```

Comment:
Både behandling og sakType henter fra 16 (behandling hentet tidligere fra 17), er det rett?

Reply:
Bra observert, ser at jeg ikke la merke til at det var en gap der og at den var plassert lenger opp, bra argument for å gå bort fra index-basert uthenting